flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/planner/functions/BuiltInFunctionTestBase.java

```diff
@@ -127,7 +126,8 @@ private static void testTableApiResult(
 
     private static void testTableApiError(Table inputTable, TableApiErrorTestItem testItem) {
         try {
-            inputTable.select(testItem.expression).execute();
+            // hasNext leads to call of eval which potentially could have runtime checks
```

Comment:
the comment is very specific for a test base of different functions, not every function might have an `eval`. how about:
`we are calling hasNext here to trigger runtime exceptions`

Reply:
yes, agree